docker/bash.sh

```diff
@@ -453,6 +453,12 @@ if [ -f "${REPO_DIR}/.git" ]; then
     fi
 fi
 
+if [[ "${DOCKER_IMAGE_NAME}" == *"ci_adreno"* && ! -z "${ADRENO_OPENCL}" ]]; then
+    DOCKER_MOUNT+=( --volume ${ADRENO_OPENCL}:/adreno-opencl)
+    DOCKER_ENV+=( --env ADRENO_OPENCL=/adreno-opencl )
+    DOCKER_ENV+=( --net=host)
```

Comment:
Why is `--nest=host` needed? We've typically avoided it in `bash.sh` in the past since its usually overly permissive

Reply:
Most of the times adb server runs on host and this enables the device access inside docker.

Reply:
it looks like net=host gets turned on already: https://github.com/apache/tvm/blob/52dbf102cdba1186e517977ee02aaa7bbe46d0df/docker/bash.sh#L346

i’d prefer we avoid setting it automatically like this so it didn’t get enabled accidentally in CI

Reply:
bash.sh is invoked from [tests/scripts/ci.py](https://github.com/apache/tvm/blob/main/tests/scripts/ci.py) and now we don't have a way of passing this flag from ci.   Let me enhance the ci script to accommodate container specific flags.